src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssProperties = sourceString
+    .split(';')
+    .map((property) => property.trim())
+    .filter((property) => property.length)
+    .map((property) => {
+      return property.split(':').map((style) => style.trim());
+    });
+
+  return Object.fromEntries(cssProperties);
```

Comment:
You can simplify and generate an object with `reduce` method instead of transform entries.